src/utils/metabase-plugins.tsx

```diff
@@ -3,8 +3,8 @@ import { MetabaseClickActionPluginsConfig } from "@metabase/embedding-sdk-react"
 import { productIdForClickActionModalAtom } from "../store/click-actions"
 import { store } from "../store"
 
-const ORDERS_TABLE_ID = 53
-const PRODUCTS_TABLE_ID = 45
+const ORDERS_TABLE_ID = 38
```

Comment:
Please use enttiy id

Reply:
as per our discussion. It's [impossible to do this right now.](https://metaboat.slack.com/archives/C01LQQ2UW03/p1746176237220079?thread_ts=1746032089.786019&cid=C01LQQ2UW03)